src/stores/elioStore.tsx

```diff
@@ -450,7 +450,7 @@ const useElioStore = create<ElioStore>()((set, get, store) => ({
       timestamp: Date.now(),
       // txnHash?: string;
     };
-    get().updateIsTxnProcessing(false);
+    // get().updateIsTxnProcessing(false);
```

Comment:
This is called when we display "Created DAO Succesfully" notification disabling the loader before the page navigates